demo/hello.ts

```diff
@@ -360,12 +360,20 @@ export class HelloApp {
         },
       },
       {
-        key: 'nested.arrays.0',
+        key: 'nested.arrays.0.item',
+        type: 'input',
+        templateOptions: {
+          label: 'Array with dots',
+        },
+        defaultValue: 'Default Value A',
+        parsers: [(value) => (value || '').toUpperCase()],
+      }, {
+        key: 'nested.arrays[1].item',
```

Comment:
I'm not in favor of this, isn't `nested.arrays.1.item` enough ?
also this make us consistent with angular form